src/main/java/frc/robot/utils/notesimulator/NoteManager.java

```diff
@@ -0,0 +1,59 @@
+package frc.robot.utils.notesimulator;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import java.util.ArrayList;
+import java.util.function.Supplier;
+
+public class NoteManager {
```

Comment:
Since this isn't a subsystem, could we not just make this a class that gets called in a simulation periodic function in RobotContainer whenever shoot is true? That way we don't need to schedule a command to shoot the note?

Reply:
I'm not sure what this would look like- since I use interpolated aim values from the shootsubsystem method, to move NoteManager to RobotContainer would I be pulling the interpolated values directly and not use data from ShootSubsystem?
Thank you!

Reply:
We can have it live wherever you want tbh, I just don't really think we need a command to call the note manager. It seems like we could have a function in note manager that is called periodically to update the simulated note if needed